pkg/controller/info/info.go

```diff
@@ -50,12 +50,25 @@ func maskUser(s, username string) string {
 	return strings.ReplaceAll(s, username, "(USER)")
 }
 
-func (c *Controller) Info(_ context.Context, _ *logrus.Entry, param *config.Param) error { //nolint:funlen
+func getCurrentUserName() (string, error) {
 	currentUser, err := user.Current()
 	if err != nil {
-		return fmt.Errorf("get a current user: %w", err)
+		return "", fmt.Errorf("get a current user: %w", err)
 	}
 	userName := currentUser.Username
+	if runtime.GOOS == "windows" {
```

Comment:
I used a runtime detection instead of compile time as I didn't think the complexity is worth the benefit.